go.mod

```diff
@@ -88,7 +88,7 @@ require (
 	github.com/meirf/gopart v0.0.0-20180520194036-37e9492a85a8
 	github.com/mutecomm/go-sqlcipher/v4 v4.4.2
 	github.com/schollz/peerdiscovery v1.7.0
-	github.com/siphiuel/lc-proxy-wrapper v0.0.0-20230516150924-246507cee8c7
+	github.com/vitvly/lc-proxy-wrapper v0.0.0-20240210131537-0f48377f5a71
```

Comment:
This repo has no license. Probably a good idea to add a MIT/Apache2 and/or move it into status-im